src/openrct2/ride/Vehicle.cpp

```diff
@@ -7273,6 +7275,16 @@ void Vehicle::UpdateAnimationAnimalFlying()
     }
 }
 
+uint8_t Vehicle::GetTargetFrame(rct_ride_entry_vehicle* vehicleEntry)
```

Comment:
Would be better to have this a static function, you are exposing this function in the header for no reason.

Reply:
I thought about this but it also `animationState +=` and it would cause a bunch of duplicate unnecessary lines.

Reply:
Even if I remove `animationState +=` there's use for this as a member function for https://github.com/OpenRCT2/OpenRCT2/pull/15398

Is it better to rewrite it from member function to static function in this PR then rewrite it from static function to member function in the follow-up PR, or keep it a member function from the get-go?